templates/core_courseformat/local/courseindex/section.mustache

```diff
@@ -0,0 +1,37 @@
+<a
+        class="courseindex-section d-block {{#current}}current{{/current}}"
```

Comment:
Weird indentation here. Not 100% sure what the coding standards are for mustache templates but it feels like the indentation should be 4 spaces here.